common.js

```diff
@@ -1621,6 +1621,166 @@ const allSoloCars = {
+
+    },
+    '86 (with TRD suspension and wheels': {
+      '2017': ['ds'],
+      '2018': ['ds'],
+      '2019': ['ds'],
+      '2020': ['ds'],
+    },
+    'Camry': {
+      'all': ['hs'],
+    },
+    'Celica FWD; NOC': {
+      'all': ['gs'],
+    },
+    'Celica All-Trac Turbo': {
+      'all': ['gs'],
+    },
+    'Celica (all, non-AWD)': {
+      '1971': ['hs'],
+      '1972': ['hs'],
+      '1973': ['hs'],
+      '1974': ['hs'],
+      '1975': ['hs'],
+      '1976': ['hs'],
+      '1977': ['hs'],
+      '1978': ['hs'],
+      '1979': ['hs'],
+      '1981': ['hs'],
+      '1982': ['hs'],
+      '1983': ['hs'],
+      '1984': ['hs'],
+      '1985': ['hs'],
+      '1986': ['hs'],
+      '1987': ['hs'],
+      '1988': ['hs'],
+      '1989': ['hs'],
+      '1991': ['hs'],
+      '1992': ['hs'],
+      '1993': ['hs'],
+      '1994': ['hs'],
+      '1995': ['hs'],
+      '1996': ['hs'],
+      '1997': ['hs'],
+      '1998': ['hs'],
+      '1999': ['hs'],
+  
+    },
+    'Celica GT': {
+      '2000': ['gs'],
+      '2001': ['gs'],
+      '2002': ['gs'],
+      '2003': ['gs'],
+      '2004': ['gs'],
+      '2005': ['gs'],
+    },
+    'Celica GTS': {
+      '2000': ['gs'],
+      '2001': ['gs'],
+      '2002': ['gs'],
+      '2003': ['gs'],
+    },
+    'Corolla': {
+      'all': ['hs'],
+    },
+    'Cressida': {
+      'all': ['hs'],
+    },
+    'Echo': {
+      'all': ['hs'],
+    },
+    'Matrix': {
+      'all': ['hs'],
+    },
+    'MR2 Turbo': {
+      'all': ['cs'],
+    },
+    'MR2 (non-turbo)': {
+      '1985': ['es'],
+      '1986': ['es'],
+      '1987': ['es'],
+      '1988': ['es'],
+      '1989': ['es'],
+      '1990': ['es'],
+      '1991': ['es'],
+      '1992': ['es'],
+      '1993': ['es'],
+      '1994': ['es'],
+      '1995': ['es'],
+    },
+    'MR2 Spyder': {
+      '2000': ['es'],
+      '2001': ['es'],
+      '2002': ['es'],
+      '2003': ['es'],
+      '2004': ['es'],
+      '2005': ['es'],
+    },
+    'MR2 Supercharged': {
+      'all': ['es'],
+    },
+    'Supra': {
+      '2020': ['bs'],
+    },
+    'Supra Turbo': {
+      '1987': ['fs'],
+      '1988': ['fs'],
+      '1989': ['fs'],
+      '1990': ['fs'],
+      '1991': ['fs'],
+      '1992': ['fs'],
+      '1993': ['bs'],
+      '1994': ['bs'],
+      '1995': ['bs'],
+      '1996': ['bs'],
+      '1997': ['bs'],
+      '1998': ['bs'],
+    },
+    'Supra (non-turbo)': {
+      '1979': ['hs'],
+      '1980': ['hs'],
+      '1981': ['hs'],
+      '1982': ['hs'],
+      '1983': ['hs'],
+      '1984': ['hs'],
+      '1985': ['hs'],
+      '1986': ['hs'],
+      '1987': ['hs'],
+      '1988': ['hs'],
+      '1989': ['hs'],
+      '1990': ['hs'],
+      '1991': ['hs'],
+      '1992': ['hs'],
+      '1993': ['fs'],
+      '1994': ['fs'],
+      '1995': ['fs'],
+      '1996': ['fs'],
+      '1997': ['fs'],
+      '1998': ['fs'],
+    'Paseo': {
+      'all': ['hs'],
+    },
+    'Prius': {
+      'all': ['hs'],
+    },
+    'Starlet': {
+      'all': ['hs'],
```

Comment:
This can also be in FSP and DP, be sure to search all of Appendix A, not just the street classes.